### PR TITLE
[physmem] Describe page size calculation.

### PR DESCRIPTION
**Pull request by MichalBlk**
Provide a description of the page size computation method.

### CODE REVIEW

File: sys/kern/vm_physmem.c [hunk 1]
```diff
@@ -77,27 +80,34 @@ static void *vm_boot_alloc(size_t n) {
...
   return begin;
 }
 
 static void vm_boot_finish(void) {
-  vm_kernel_end = (vaddr_t)align((void *)vm_kernel_end, PAGESIZE);
   vm_boot_done = true;
 }
 
+static inline size_t pg_compute_size(paddr_t pa) {
+  /* Take the highest number of the form 2^k * PAGESIZE
```
Comment (hadarai) [inline sys/kern/vm_physmem.c:106]: Do I understand correctly that you don't explain here what is `PM_NQUEUES - 1`?
(only second argument of `min`)

**Comment by cahirwpz** [comment]
Is `bug` label still valid for this PR? Please resolve conflicts.

**Comment by cahirwpz** [comment]
Please resolve conflicts.

**Comment by MichalBlk** [comment]
Conflicts have been resolved.

**Review by wiklam — APPROVED** [review]

**Review by hadarai — APPROVED** [review]

**Comment by cahirwpz** [comment]
@MichalBlk What do we do about this PR?

**Comment by MichalBlk** [comment]
@cahirwpz The PR has been updated. Now, it simply adds a description to the page size calculation logic.

**Review by cahirwpz — APPROVED** [review]

**Pull request merged**